.gitlab-ci.yml

```diff
@@ -140,8 +140,6 @@ doc-tests:v1.6:
 
 pages:
   stage: documentation
-  dependencies:
```

Comment:
I think it is still required, no? I think we cannot assume that gource is always running ...

Reply:
the jobs are technically independent, and I wanted the gource to be built only from "released" stuff on master (stuff in develop might be messy/reorganized).

Reply:
the jobs are not really independent - like the deployment of the docs should not happen if gource has failed

Reply:
ah okay, that makes sense